CMakeLists.txt

```diff
@@ -173,6 +177,17 @@ if(BUILD_OPENCV_CAMERA_PLUGIN)
     list(APPEND OPENCV_MODULES_USED ${OPENCV_CAMERA_EXTRA_LIBS})
 endif()
 
+if(BUILD_VIDEOTRACKER_PLUGIN)
+    set(VIDEOTRACKER_EXTRA_LIBS
+        opencv_imgproc
+        opencv_features2d
+        opencv_calib3d
```

Comment:
Do you know if these libs are the same in OpenCV 3? For instance, I know highgui's video capture got split out into `videoio` (see the lines above)